lib/tasks.ex

```diff
@@ -1,18 +1,23 @@
 defmodule Tasks do
-  @moduledoc """
-  Documentation for `Tasks`.
-  """
+  def frequencies(string) do
+    list =
+      string
+      |> String.split()
+      |> Enum.map(fn word -> String.downcase(word) end)
```

Comment:
This would be better this way

```elixir
     string
      |> String.downcase()
      |> String.split()
```

In your solution you do a call to downcase on every word which is needless.